test/Tool/known8-llvm.opt

```diff
@@ -0,0 +1,14 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -print-counterexample=false  %s > %t 2>&1
+; RUN: FileCheck %s < %t
+
+; CHECK: LGTM
```

Comment:
This test needs to be changed or removed-- the optimization is independent of the known bit information.